docs/sources/experiments/cohort-labels.md

```diff
@@ -0,0 +1,493 @@
+# Cohort and Label Deep Dive
+
+This document is intended at providing a deep dive into the concepts of cohorts and labels as they apply to Triage. For context, reading the [Triage section of the Dirty Duck tutorial](https://dssg.github.io/dirtyduck/#sec-4-2) may be helpful before reading this document.
+
+## Temporal Validation Refresher
+
+Triage uses temporal validation to select models because the real-world problems that Triage is built for tend to have a strong temporal correlation.  Picking a date range to train on and a date range afterwards to test on ensures that we don't leak data from the future into our models that wouldn't be available in a real-world deployment scenario. Because of this, we often talk in Triage about the *as-of-date*: all models trained by Triage are associated with an *as-of-date*, which means that all the data that goes into the model **is only included if it was known about before that date**. For more on temporal validation, see the [relevant section in Dirty Duck](https://dssg.github.io/dirtyduck/#sec-4-2-2-1).
```

Comment:
The models are associated to an `as-of-date`, but every *row* inside the matrices is linked to a set of *as of dates* (including the one linked to the model)

Reply:
I'm also trying to figure out how to phrase this here in the doc without going down a rabbit hole. I'll get back to you.